python/py-upt-macports/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+set git_hash        20b55dcc6a06f48da8a22c415bc74315dda5cb2e
+github.setup        macport upt-macports ${git_hash}
+version             0.1
```

Comment:
I would add the date to the version string. It's likely that we'll not increase the version anytime soon, but by adding the date to ```version``` it will still be recognized as a newer version. At least I think that's the way we typically do it (at least what I do for ```py-spyder-devel```... @mojca?

Reply:
it looks like other ports do something along the lines of ```version      YYYYMMDD-[string range ${github.version} 0 7]```, which has the advantage that the version always increases.

Reply:
We can make an arbitrary decision here. One option is to actually tag every version we want to release in upt-macports. One option is to simply increase the revision after any update (just until this stabilizes). One option is to suffix the version with the date (and optionally with a short git tag, but I don't see a reason to overcomplicate stuff). I don't care what is used, let's just not make our lives miserable due to pointless discussion and overengineered solutions, just pick one way and go with it (don't increase the epoch to increase the version though :).

Reply:
As of now, I have kept it to `version-YYYYMMDD`